create-snowpack-app/app-template-svelte/public/index.html

```diff
@@ -2,14 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="/favicon.ico" />
+    <base href="%PUBLIC_URL%" />
+    <link rel="icon" href="favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Web site created using create-snowpack-app" />
     <title>Snowpack App</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <script type="module" src="/_dist_/index.js"></script>
+    <script type="module" src="_dist_/index.js"></script>
```

Comment:
Ah right, i just saw this issue myself in a project. Didn't realize that it would be the same issue for GH pages. 

Instead of setting `<base>` in these templates, can you set PUBLIC_URL directly in all places that use it?

```diff
+ <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />

+ <script type="module" src="%PUBLIC_URL%/_dist_/index.js"></script>
```

And can you do this for all of the templates, not just the svelte one?

Reply:
One of the main things I ran into with our `%PUBLIC_URL%` vs CRA's version is we add a trailing slash to the path. This example `%PUBLIC_URL%/favicon.ico` results in something like `/foobar//favicon.ico`. The same thing happens with the `base` tag.

I personally always like to use the `base` tag and not have to mess with the imports.

Reply:
ah right, that is annoying. hmm... may be to late to change

Reply:
Ok, I will change all the templates to use PUBLIC_URL but not use the base html element.

Reply:
I have updated al of the templates. @FredKSchott @stramel